staramr/blast/results/pointfinder/BlastResultsParserPointfinder.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, file_blast_map, blast_database, pid_threshold, plength_thresh
 
     def _create_hit(self, file, database_name, blast_record):
         logger.debug("database_name=%s", database_name)
-        if database_name == '16S_rrsD':
+        if (database_name == '16S_rrsD') or (database_name == '23S'):
```

Comment:
Since we'd added an additional check for `23S`, let's also add a test case for `23S` for campy (sorry about saying you should only include one test before). So, you'd do the same thing as for the case with `gyrA` (that is, copy the `23S` file from the pointfinder database, make a mutation in the correct location, and add a test case).